README.md

```diff
@@ -226,12 +223,6 @@ If you have installed an SSD and a HDD in your T430, you can gain the advantages
 	-> rebuild cache short form
         sudo touch /System/Library/Extensions && sudo kextcache -u /
 
-
-#### GENERATE VANILLA CONFIG.PLIST (if you need it)
-
-	on /usr/local/bin/clover-genconfig > config.plist
-	execute and take the xml
-
 #### HOW TO FLASH T430 BIOS (if you want to install an 1300AC Wi-Fi …)
```

Comment:
You should probably add something here about the [ivyra1n BIOS mods](https://github.com/n4ru/1vyrain) since that unlocks everything you need to really tune your hack and then some.